### PR TITLE
Adds example for embedded database + command line database generation

### PR DESCRIPTION
**Pull request by sidferreira**

### CODE REVIEW

**Comment by sidferreira** [comment]
@Stophface @devnrj07

**Comment by Stophface** [comment]
@sidferreira I'll have a look at it asap.

File: examples/native/scripts/generateEmbeddedDatabase.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* this script needs @babel/node to run, check package.json script session */
+import * as fs from 'fs';
+import databasePromise from '../src/database/embeddedInit';
+
+import { generate100 } from '../src/model/generate';
```
Comment (Stophface) [inline examples/native/scripts/generateEmbeddedDatabase.js:5]: @sidferreira  where is the `generate100` function? It's not in here `https://github.com/Nozbe/WatermelonDB/tree/master/src/Model`, yet you import it from `import { generate100 } from '../src/model/generate';`

File: examples/native/scripts/generateEmbeddedDatabase.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  if (!fs.existsSync('android/app/src/main/assets')) {
+    fs.mkdirSync('android/app/src/main/assets');
+  }
+  fs.copyFileSync(dbName, `android/app/src/main/assets/${dbName}`);
+  fs.unlinkSync(dbName);
+  console.log('Database Generated!');
+}
+
+generateEmbeddableDatabase();
```
Comment (Stophface) [inline examples/native/scripts/generateEmbeddedDatabase.js:27]: @sidferreira Am I correct in assuming that this creates a database and then moves it to location in the database filesystem, where an embedded database would be stored?

File: examples/native/scripts/generateEmbeddedDatabase.js (hunk 1 above)
Comment (Stophface) [reply examples/native/scripts/generateEmbeddedDatabase.js:5]: @sidferreira Where is it?

Comment (sidferreira) [reply examples/native/scripts/generateEmbeddedDatabase.js:5]: @Stophface ping me in twitter

Comment (sidferreira) [reply examples/native/scripts/generateEmbeddedDatabase.js:5]: but if you check Root.js you will see that the generate code was already there in Model

**Comment by Stophface** [comment]
Thanks for your example @sidferreira I think Ive got it running now! <3

**Comment by sidferreira** [comment]
Go to https://github.com/Nozbe/WatermelonDB/issues/774#issuecomment-1805090189 for latest documentation

**Pull request closed without merging**